### ISSUE TITLE
[ FEATURE REQUEST ] - Natsuki asks the player's gender

### ISSUE THREAD

**Issue by disownxd** [Just-Natsuki-Team/NatsukiModDev#504]
# Overview:

As a girl, I think it would be appropriate for Natsuki to ask what gender the player identify as, especially in higher affection levels. After all, I would like to know the gender of the person that I'm slowly falling for.

## Must haves:

· Natsuki asking delicately if the player wants to share that information with her, and also the option to say that you don't feel comfortable talking about it;

· Options like: male, female, neither of them, or "I'm still figuring it out";

· Natsuki could have a different kind of reaction depending on the option chosen;

· Natsuki could ask what pronouns do you prefer;

· An option to change pronouns or gender later;

## Could haves:

· Natsuki being a little flustered or confused if the option chosen is anything different form male (since it's the "default" in the vanilla DDLC);

**Comment by RaionArt** [comment]
Hello @disownxd and thank you for raising an issue with us! 

After discussing with the team we came to the conclusion that we would like to avoid topics referencing genders, as they sadly turned out to be quite political and are very emotional to some people.

As for the techincal view it wouldn't seem very preferable neither. At the moment Natsuki never uses any pronouns as she is always refering to the player with their name or simply with "you". If we would ever include dialogue which uses pronouns we would need to make sure to create systems to make Natsuki always use the right pronouns. We fear we could offense someone if some mistakes happen. Also more included systems can result in more bugs.

To sum it up, we don't feel the effort involved in introducing a way to track this information (via topic) and using it can be justified given how gender plays little to no role in JN at all, at least for the foreseeable future: especially at a time when our main focus is on content that has broad appeal, while this feels more niche.

Thank you for your attention and understanding.

**Comment by disownxd** [comment]
Aw, that's a shame, but I understand. Thanks for the explanation, anyway.
